201-zerto-zca-vpn/azuredeploy.parameters.json

```diff
@@ -7,21 +7,24 @@
      },
      "localAddressPrefix":{  
```

Comment:
for parameters that don't use GEN values, move them to defaultValues in the template

Reply:
done